libs/styles/styles.css

```diff
@@ -10,222 +10,235 @@
  * governing permissions and limitations under the License.
  */
 
-@import url('variables.css');
-@import url('typography.css');
```

Comment:
I'm confused about why the line diff for `styles.css` is so large

Reply:
Yeah not sure how but there were extra spaces in from of nearly everything, fixed. Nice catch 👍🏻